src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  margin-right: auto;
+  margin-left: 50px;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  padding-right: 50px;
+}
+
+.nav {
+  list-style: none;
+}
+
+.nav__list a {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  margin-left: 16px;
+}
+
+.nav__list li a {
```

Comment:
yeah, you cannot simply remove the tags from here, because you are styling the `a` tag here. if you just remove the tags you will no longer be styling the link, you will be styling the list (`ul`). you need to give your links classes and use those classes instead of this combined selector